server/server.go

```diff
@@ -98,6 +107,27 @@ func (s *Server) ConnectionCount() int {
 	return cnt
 }
 
+type clientInfo struct {
+	clientID  uint32
+	user      string
+	host      string
+	sessionID uint32
+}
+
+func (s *Server) getClientsInfo() []clientInfo {
```

Comment:
Seems that the implementation of this function only lists clients using X Protocol. It would be better to rename the function for that.

Reply:
It is a little wired that there is a function for the new protocol only without a mirror function for the old one or for the both in server.go. I think maybe we need to get mysql clients info in the future, when I design this function. :) OK, I'll change these code.

Reply:
Done.